config/config.yaml

```diff
@@ -25,3 +25,6 @@ CR_TO_SYNC:
 # Applications to be deployed on the cluster
 APPS:
   - pipeline_service # pipeline_service sets up Pipeline Service on the cluster.
+
+# kcp version
+KCP_VERSION: v0.8.0
```

Comment:
Maybe I would use the following structure:
```
version:
  kcp: v0.8.0
```
We can then add more version/dependencies there in the future if necessary.

Reply:
Sounds good. I have made these changes.